CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 Changelogs for versions not listed here can be found at https://github.com/DataDog/dd-trace-py/releases
 
+---
+
+## 2.17.2
+
+
+### Bug Fixes
+
+- ASM: This fix ensures that common patches for exploit prevention and sca are only loaded if required, and only loaded once.
```

Comment:
Aren't the notes supposed to look have sub-items per category like the ones below?